test/rules/jsx-whitespace-literal/test.tsx.lint

```diff
@@ -0,0 +1,29 @@
+<div>
+    <p>Some Text</p>
+    <hr></hr>
+    Some Text&nbsp;&nbsp;
+    ~~~~~~~~~~~~~~~~~~~~
+    <select>
+~~~~ [0]
+    <option></option>
+    </select>
+    <button type="button">Some Text</button>
+    <br/>
+    Some Text&nbsp;
+    ~~~~~~~~~~~~~~~
```

Comment:
same here, this failure text range is too wide

Reply:
Will fix it.

Reply:
Done